styles.css

```diff
@@ -0,0 +1,139 @@
+
+body {
+  width: 100vw;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+header {
+  height: 48px;
+  width: 100vw;
+  margin: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+nav {
+  height: 48px;
+  width: 100vw;
+  font-size: 17px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: 0.3s;
+  margin: 0;
+  padding: 0% 2.5%;
+  background-color: rgba(60, 58, 57, 1);
+  color: white;
+  font-weight: 500;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.logo {
+  width: 100px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+
+#menu-button {
+  /* height: 100%; */
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2%;
+  cursor: pointer;
+}
+
+a:link {
+  text-decoration: none;
+  color: white;
+}
+
+a:visited {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  text-decoration: none;
+  color: white;
+}
+
+a:active {
+  text-decoration: none;
+  color: white;
+}
+
+#first {
+  height: 100vh;
+  padding: 168px 24px 120px 24px;
+  color: white;
+  width: 100vw;
+  background: no-repeat url("./images/fs-back.png");
+  background-position: 75vw 110%;
+  background-color: rgba(28, 26, 25, 1);
+}
```

Comment:
- I currently cannot see this background image. I guess this is because you are referencing the wrong URL path, your image folder starts with a capital `I`. Please fix this. 😄 
- Please fix this issue for all images also 🙏 
- I noticed you are using the wrong background image for this section, please check the Figma design and export the correct background image for the mobile version. 👍 👍

Reply:
Good job following the advice of the previous reviewer